### PR TITLE
live preview support 1.0 and 2.0

### PR DESCRIPTION
**Pull request by harshithad0703**
live preview support 1.0 and 2.0

- 1.0 = when management token is provided, host will be previous host
- 2.0 = when preview token is provided, host will be new host

### CODE REVIEW

**Comment by sunil-lakshman** [comment]
> @harshithad0703 , we have already released v3.18.1. What should be the version here @ishaileshmishra cc : @nadeem-cs @sunil-lakshman ?

@harshithad0703 This is a new feature, make the version v3.19.0

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Change log
 
+### Version: 3.18.1
```
Comment (sunil-lakshman) [inline CHANGELOG.md:3]: Change here also

**Comment by abhinav-from-contentstack** [comment]
@sunil-lakshman I think only live preview host and config keys have been updated (live preview feature already exists in the current version), so patch version update should be fine `v3.18.2`

**Review by sunil-lakshman — APPROVED** [review]
Approved

**Comment by abhinav-from-contentstack** [comment]
updated version to `v3.19.0` as there is a breaking change that might affect live preview for references

**Pull request merged**